pages/api/studies/key/[studyKey].js

```diff
@@ -0,0 +1,15 @@
+import dbConnect from '@lib/db-connect'
+import Study from '@models/study'
+
+export default async function handler(req, res) {
+  await dbConnect()
+
+  const { studyKey } = req.query
+
+  try {
+    const studies = await Study.findOne({ key: studyKey })

```

Comment:
Talked about this in a previous comment but we should probably only use the ObjectId as an identifier. Also might make more sense for this endpoint to live on '/api/studies/<ObjectId>' and have the adminName endpoint live on something like '/api/studies/user/[adminName].js'

Reply:
Yeah we should definitely give a bit more thought to endpoint naming conventions before we get much further with implementation. This PR is just to have a working example of the API, mongoose, useSWR, etc. to go off of.